### PR TITLE
WIP - data clean up

### PR DESCRIPTION
**Pull request by marktani**
#8 
#7 
#6 

Don't merge yet!

### CODE REVIEW

**Comment by marktani** [comment]
I generated a new file for the country data that uses the correct country code `ccio` instead of `cca3`. I had to clean up a lot of data, for example in the Wikipedia data, Spain and Andorra are combined, or Switzerland and Liechtenstein. In these cases, I removed the "smaller" country from the name, like "Liechtenstein".

I did some other clean up like renaming country names to fit the values from `countries.json` from the `countries` repo, for example from The Gambia to Gambia.

Current shape of the new data:

```
[
  {
    "country": "afg",
    "total1990": "2.546",
    "total2005": "1.063",
    "total2017": "11.422",
    "percent2017": "0.03",
    "percentage20171990": "448.6",
    "emissionIntensity2017": "18",
    "emissionPerCapita2017": "0.3"
  },
...
]
```

Expected shape:

```
{
  "afg": {
    "total1990": "2.546",
    "total2005": "1.063",
    "total2017": "11.422",
    "percent2017": "0.03",
    "percentage20171990": "448.6",
    "emissionIntensity2017": "18",
    "emissionPerCapita2017": "0.3"
  },
```

**Comment by marktani** [comment]
I fixed #7 and #8, I am stuck with #6. Can someone show me how to add a random React component to the map, so I can build the legend?

**Comment by marktani** [comment]
pls don't judge me 😅

image


thanks @divyenduz for the help!

**Comment by marktani** [comment]
Ready to merge 🙏

File: components/Map/Map.tsx [hunk 1]
```diff
@@ -51,18 +107,56 @@ const defaultViewPort: ViewState = {
...
+        width: 10px;
+        height: 10px;
+        border-radius: 50%;
+        float: left;
+      }
+    `}</style>
+  </div>
+)
+
+const legend = () => (
```
Comment (divyenduz) [inline components/Map/Map.tsx]: This can move to its own file as a component, other than this amazing work 🙌

Comment (marktani) [reply components/Map/Map.tsx]: might pick this up later!

**Pull request merged**